README.md

```diff
@@ -13,6 +13,23 @@ Please see [action.yml](action.yml)
 
 ## Example Usage
 
+You need two things: 
+ 1. an "ECHIDNA" file
```

Comment:
```suggestion
 1. an "ECHIDNA" manifest file
```
Echidna calls it manifest file: https://github.com/w3c/echidna/wiki/Preparing-your-document#manifest-file